src/sentry/monitors/endpoints/organization_monitor_details.py

```diff
@@ -23,11 +25,18 @@
 from sentry.constants import ObjectStatus
 from sentry.models.rule import Rule, RuleActivity, RuleActivityType
 from sentry.models.scheduledeletion import RegionScheduledDeletion
-from sentry.monitors.models import Monitor, MonitorEnvironment, MonitorStatus
+from sentry.monitors.models import (
+    CheckInStatus,
+    Monitor,
+    MonitorCheckIn,
+    MonitorEnvironment,
+    MonitorStatus,
+)
 from sentry.monitors.serializers import MonitorSerializer
 from sentry.monitors.utils import create_alert_rule, update_alert_rule
 from sentry.monitors.validators import MonitorValidator
 
+from ..constants import MAX_TIMEOUT, TIMEOUT
```

Comment:
Should we be using relative imports like this @asottile-sentry?

Reply:
I personally don't like them because they make codemods much more difficult but technically there's nothing wrong with them.

the main benefit to them is for vendoring which we aren't doing